bin/cake

```diff
@@ -2,6 +2,17 @@
 
 var path = require('path');
 var fs   = require('fs');
-var lib  = path.join(path.dirname(fs.realpathSync(__filename)), '../lib');
 
-require(lib + '/coffeescript/cake').run();
+var potentialPaths = [
+  path.join(process.cwd(), 'node_modules/coffeescript/lib/coffeescript'),
+  path.join(process.cwd(), 'node_modules/coffeescript/lib/coffee-script'),
+  path.join(process.cwd(), 'node_modules/coffee-script/lib/coffee-script'),
+  path.join(path.dirname(fs.realpathSync(__filename)), '../lib/coffeescript')
+];
+
+for (var i = 0; i < 4; i++) {
```

Comment:
I was trying to be optimized, though I can see how this comes across as overkill. I toyed with a series of if statements to avoid even the unnecessary path manipulation lines, but testing didn't show a difference. I can make this more idiomatic if you think it's silly to hard-code the length to save 1ms.